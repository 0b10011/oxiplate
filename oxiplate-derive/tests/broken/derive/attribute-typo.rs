use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplatee = ""]
struct Data;

fn main() {}
