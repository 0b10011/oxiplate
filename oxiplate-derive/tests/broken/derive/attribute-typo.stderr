error: proc-macro derive panicked
 --> tests/broken/derive/attribute-typo.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: not implemented: Must specify an attribute

error: cannot find attribute `oxiplatee` in this scope
 --> tests/broken/derive/attribute-typo.rs:4:3
  |
4 | #[oxiplatee = ""]
  |   ^^^^^^^^^ help: a derive helper attribute with a similar name exists: `oxiplate`
